cloud_governance/common/clouds/aws/athena/AthenaOperations.py

```diff
@@ -0,0 +1,79 @@
+from datetime import datetime
+
+import boto3
+import typeguard
+from pyathena import connect
+from pyathena.pandas.cursor import PandasCursor
+
+from cloud_governance.common.logger.init_logger import logger
+from cloud_governance.common.logger.logger_time_stamp import logger_time_stamp
+from cloud_governance.main.environment_variables import environment_variables
+
+
+class AthenaOperations:
```

Comment:
I think we should split it to 2 classes or inheritance from the same interface:
AthenaOperations
PyAthenaOperations

When you have more queries like delete, update, insert. it will cause to confusing.

Reply:
@ebattat Adde the AbstarctAthenaOperations for the PyAthenaOperations, Boto3ClientAthenaOperations